talentprotocol/mvp
# Add paragraph links

## Summary
For the ability to add links on modals, we needed ones that give more contrast than the ones we have right now.
Besides that we added a different hover to the earn menu dropdown items

## Notion link
https://www.notion.so/talentprotocol/Earn-Tab-Dropdown-db4e8c1de7194da5bdb181fd38a0c515

## How to test?
Open the earn menu dropdown and check if the icons are purple on hover
Open the quests menu and check for the new paragraph links
